### DISCUSSION TITLE
Confusion in paper

### DISCUSSION THREAD

**Discussion by pieceofGit** [yandex-research/rtdl-revisiting-models#17] [Q&A] [answered]
Hi, thought it was possible to ask here. In chapter 4.4 Comparing DL models you compare first the models on the datasets without using an ensemble structure for ResNet and FT-Transformer. NODE has a significantly high performance on a few datasets. Then you compare the three all using ensemble structures and say that the results indicate that FT-Transformer and ResNet benefit more from ensembling, but then say that the gap between ResNet and NODE is significantly reduced with ensembling. I would think the gap has increased between NODE and ResNet based on the wording. 
Br. Fjölnir

**Comment by Yura52** [comment, marked as answer]
Hi! By "the gap is reduced" we mean "the gap became *smaller*". Which is in line with the statement "FT-Transformer and ResNet benefit more from ensembling", since "more benefit" leads to "smaller gaps" on the problems where NODE is superior to ResNet.
Does this help?

**Reply by pieceofGit** [reply]
Yes, thanks
